infrastructure/main.tf

```diff
@@ -1,3 +1,32 @@
 provider "azurerm" {
   version = "=1.22.1"
 }
+
+locals {
+  s2s_rg              = "rpe-service-auth-provider-${var.env}"
+  s2s_url             = "http://${local.s2s_rg}.service.core-compute-${var.env}.internal"
+
+  vaultName           = "bulk-scan-${var.env}"
+}
+
+data "azurerm_key_vault" "s2s_key_vault" {
+  name                = "s2s-${var.env}"
+  resource_group_name = "${local.s2s_rg}"
+}
+
+data "azurerm_key_vault_secret" "s2s_secret" {
+  key_vault_id = "${data.azurerm_key_vault.s2s_key_vault.id}"
+  name      = "microservicekey-bulk-scan-payment-processor"
+}
+
+# Copy s2s secret from s2s key vault to bulkscan key vault
+data "azurerm_key_vault" "bulk_scan_key_vault" {
+  name = "${local.vaultName}"
```

Comment:
mis-alignment here too

Reply:
p.s. can avoid using "local" - just stick in vault name here directly

Reply:
> just a name. but since that needs changing - might as well apply styling 🙂

renamed and applied style changes